src/sortable.coffee

```diff
@@ -32,14 +37,14 @@ class Sortable extends DragAndDrop
 
   enable: ->
     unless @enabled
-      @$el.on "dragstart", @options.items, $.proxy(this, "_handleDragstart")
-      @$el.on "dragover", @options.items, $.proxy(this, "_handleDragover")
+      @$el.on "dragstart", $.proxy(this, "_handleDragstart")
```

Comment:
I'm wondering if we need to change the binding on this event specifically since we only want to listen for draggable item drag start, not other item drag start?

Reply:
Asking because I'm wondering if we could avoid needing extra logic in `_handleDragstart` by not changing anything here, if it's not actually needed.

Reply:
looks like this one can be reverted

Reply:
thanks